2-2-7.py

```diff
@@ -0,0 +1,22 @@
+'''
+Тимур и Руслан пытаются разделить фронт работы по курсу "Python для профессионалов". Для этого они решили сыграть в камень, ножницы и бумагу. Помогите ребятам бросить честный жребий и определить, кто будет делать очередной модуль нового курса.
+
+Формат входных данных
+На вход программе подаются две строки текста, содержащие слова "камень", "ножницы" или "бумага".
+На первой строке записан выбор Тимура, на второй – выбор Руслана.
+
+Формат выходных данных
+Программа должна вывести результат жеребьёвки, то есть кто победит: Тимур, Руслан или же они сыграют вничью.
+
+Примечание. Правила игры стандартные: камень побеждает ножницы, но проигрывает бумаге, а ножницы побеждают бумагу.
+'''
+
+Tim, Rus = input(), input()
```

Comment:
better names would be `tim_choice`, and `rus_choice` - avoid using Capitalized variable names.

Reply:
it very strange. I think that we are corrected it together earler.